build/config/RepoUtilData.json

```diff
@@ -1,26 +1,9 @@
 ﻿{
   "fixedPackages": {
-    "Microsoft.VisualStudio.Language.Intellisense": "14.3.25407",
     "Microsoft.VSSDK.BuildTools": [ "14.3.25420", "15.0.25929-RC2" ],
     "Newtonsoft.Json": "9.0.1",
     "System.Reflection.Metadata": "1.0.21",
     "System.Collections.Immutable": "1.1.36",
-    "Microsoft.VisualStudio.CoreUtility": "14.3.25407",
-    "Microsoft.VisualStudio.Diagnostics.PerformanceProvider": "14.3.25420-alpha",
-    "Microsoft.VisualStudio.Editor": "14.3.25407",
-    "Microsoft.VisualStudio.Language.NavigateTo.Interfaces": "14.3.25407-alpha",
-    "Microsoft.VisualStudio.Imaging": "14.3.25407",
-    "Microsoft.VisualStudio.Imaging.Interop.14.0.DesignTime": "14.3.25407",
-    "Microsoft.VisualStudio.ImageCatalog": "14.3.25407",
-    "Microsoft.VisualStudio.Shell.Immutable.10.0": "10.0.30319",
-    "Microsoft.VisualStudio.ComponentModelHost": "14.3.25407-alpha",
-    "Microsoft.VisualStudio.Text.UI.Wpf": "14.3.25407",
-    "Microsoft.VisualStudio.Text.Logic": "14.3.25407", 
-    "Microsoft.VisualStudio.Text.Data": "14.3.25407", 
-    "Microsoft.VisualStudio.Text.UI": "14.3.25407",
-    "Microsoft.VisualStudio.Utilities": "14.3.25407",
-    "Microsoft.VisualStudio.Validation": "14.1.111",
-    "Microsoft.VisualStudio.Threading": "14.1.111",
```

Comment:
Prior to this change there were two versions of all of these NuGet packages used in our repo:

- Dev14
- Dev15

This entry in the config file essentially told RepoUtil two things:

- This difference is intentional, don't count it as an error
- If you're asked to upgrade MS.VS.Text.Logic to a new version, do not upgrade anything that is marked as Dev14

Now that there is a single version, none of these entries are necessary.